conductor.go

```diff
@@ -91,11 +78,16 @@ func (c *Conductor) Play(ctxMain context.Context) error {
 
 	select {
 	case err := <-errs:
-		log.Printf("Error occured in a player: %s\n", err.Name)
-		return err
+		// Ensure we inform all other players to shutdown
+		cancel()
+		// Handle the error once (logging an error is handling it once,
+		// and returning it is a second action on the same error).
+		return fmt.Errorf("Error occured in a player: %s", err.Name)
 	case <-timedCtx.Done():
-		log.Println("Pool stopped after timeout")
-		return nil
+		// If this times out, then players aren't shutting down correctly.
+		// We need to inform the users of this library that one or more of their
+		// players hasn't shutdown when the context's done channel was closed.
+		return errors.New("Forcing shutdown of conductor, not all players may have shutdown")
```

Comment:
I don't think this should be an error, because even if it shutdown because of the timeout, it did shutdown as expected.

However, in some ways it does make sense to return an error. My only condition would be that the error is a custom type (or constant) that allows the caller to use `errors.Is()` to ignore it if necessary.